libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -178,7 +178,7 @@ const AP_Param::GroupInfo AC_PrecLand::var_info[] = {
     // @Param: OPTIONS
     // @DisplayName: Precision Landing Extra Options
     // @Description: Precision Landing Extra Options
-    // @Bitmask: 0: Moving Landing Target, 1: Allow Precision Landing after manual reposition 
+    // @Bitmask: 0: Moving Landing Target, 1: Allow Precision Landing after manual reposition, 2: Do not slow vertical speed in final descend phase
```

Comment:
nitpick: maybe we could word this in a positive way?  E.g. "Maintain high speed in final descent".

Also I think "descent" is better than "descend".